tests/unit/accountability_api/v2/test_data.py

```diff
@@ -67,11 +67,14 @@ def test_ListDataTypes_get(test_client: FlaskClient):
 
     # ASSERT
     assert response.status_code == 200
-    assert len(data) == 3
```

Comment:
This seems to catch if data is empty. Not sure if your next assert block catches that? Is that important to catch?